cmd/cl-adm/cmd/create/create_peer.go

```diff
@@ -58,6 +60,7 @@ func (o *PeerOptions) AddFlags(fs *pflag.FlagSet) {
 		"The log level. One of fatal, error, warn, info, debug.")
 	fs.StringVar(&o.ContainerRegistry, "container-registry", config.DefaultRegistry,
 		"The container registry to pull the project images. If empty will use local registry.")
+	fs.StringVar(&o.Version, "version", "latest", "The version of the project images.")
```

Comment:
nit: I think it's called an image `tag` and not a `version`. Using `release tag` might also be ok